charts/base-cluster/values.yaml

```diff
@@ -56,7 +56,7 @@ global:
         app.kubernetes.io/component: storage
         app.kubernetes.io/part-of: nfs-server-provisioner
     backup:
-      condition: "{{ .Values.backup.enabled }}"
+      condition: "{{ not (empty .Values.backup.backupStorageLocations) }}"
```

Comment:
Is this `not empty` really needed?

I'm unsure. We haven't used it anywhere else, but i think it also won't hurt?

Reply:
Alright, was cleared. It's ok